sechub-solution/docker/install-java/install-java.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+print_error() {
+    message="$1"
+
+    echo "$message" 1>&2
+}
+
+JAVA_DISTRIBUTION="$1"
+JAVA_VERSION="$2"
+JAVA_RUNTIME="$3"
+
+JAVA_DIR="/opt/java"
+
+if [ -z "$JAVA_DISTRIBUTION" ]
+then
+    print_error "ERROR: No Java distribution provided!" 1>&2
```

Comment:
The `1>&2` is already contained in `print_error` function and can be omitted in the calls.
```suggestion
    print_error "ERROR: No Java distribution provided!"
```